apps/src/javalab/JavalabEditor.jsx

```diff
@@ -35,6 +35,8 @@ const style = {
   }
 };
 
+const darkTabColor = '#282c34';
```

Comment:
Suggestion: Move this to a `darkBackground` or `darkMode` style i.e. 
```
darkBackground: {
  backgroundColor: '#282c34`
}
```
Then merge the styles like you do in JavalabConsole.jsx. This would allow us to easily add additional styling in the future and be sure it will be applied everywhere that is relevant.